modules/installation-osp-accessing-api-floating.adoc

```diff
@@ -0,0 +1,71 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_openstack/installing-openstack-installer.adoc
+// * installing/installing_openstack/installing-openstack-installer-custom.adoc
+// * installing/installing_openstack/installing-openstack-installer-kuryr.adoc
+
+[id="installation-osp-accessing-api-floating_{context}"]
+
+= Enabling API access with floating IP addresses
+
+Make {product-title} API endpoints accessible by attaching two floating IP (FIP) addresses to them: one for the API load balancer (`lb FIP`), and one for OpenShift applications (`apps FIP`).
+
+[IMPORTANT]
+The load balancer FIP is also used in the `install-config.yaml` file.
+
+.Procedure
+
+. Using the OpenStack CLI, create a new external network:
++
+----
+$ openstack floating ip create <external network>
+----
+
+. Add a record that follows this pattern to your DNS server:
++
+[source,dns]
+----
+api.<cluster name>.<base domain>  IN  A  <lb FIP>
+----
++
+[NOTE]
+====
+If you don't control the DNS server, you can add the record to your `/etc/hosts` file instead. This action makes the API accessible to you only, which is not suitable for production deployment, but does allow installation for development and testing.
+====
+// Break the rest of this into its own module that follows OpenShift installation, then?
```

Comment:
@tomassedovic Thinking of order again, does it make sense to move this portion of this procedure after the installer's run, or am I wrong?